test/src/module/command/restoreTest.c

```diff
@@ -403,12 +403,12 @@ testRun(void)
         storagePathCreateP(storagePgWrite(), NULL);
 
         // -------------------------------------------------------------------------------------------------------------------------
-        TEST_TITLE("error on postmaster.pid is present");
+        TEST_TITLE("error on when pg appears to be running");
```

Comment:
Silly typo here (remove "on") - should just be "error when pg appears to be running"

Reply:
Haha, way to propagate typo. Fixed.